I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 4525, in script
    call screen openletter
ScriptError: could not find label 'afteropenletter'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "C:\Users\user\Videos\game-project\renpy\bootstrap.py", line 316, in bootstrap
    renpy.main.main()
  File "C:\Users\user\Videos\game-project\renpy\main.py", line 578, in main
    run(restart)
  File "C:\Users\user\Videos\game-project\renpy\main.py", line 143, in run
    renpy.execution.run_context(True)
  File "C:\Users\user\Videos\game-project\renpy\execution.py", line 908, in run_context
    context.run()
  File "game/script.rpy", line 4525, in script
    call screen openletter
  File "C:\Users\user\Videos\game-project\renpy\script.py", line 898, in lookup
    raise ScriptError("could not find label '%s'." % str(original))
ScriptError: could not find label 'afteropenletter'.

Windows-8-6.2.9200
Ren'Py 7.3.5.606
Blue Blood Red blood 1.0
Mon Apr 12 20:31:09 2021
